packages/asc-ui/src/components/InfoBanner/InfoBannerStyle.ts

```diff
@@ -0,0 +1,93 @@
+import styled, { css, Theme } from '@datapunt/asc-core'
+import {
+  focusStyleOutline,
+  svgFill,
+  themeColor,
+  themeSpacing,
+} from '../../utils'
+import Button from '../Button'
+
+export type Level = 'normal' | 'attention' | 'error'
```

Comment:
Do we want to go with `'normal' | 'attention' | 'error'`? Or perhaps `'info' | 'warning' | 'error'`?

Reply:
Personally I think attention and warning are closely related to each other and might be confusing.